ParquetUnitTests/AssemblyUnitTest.cs

```diff
@@ -1,12 +1,29 @@
+using System.Linq;
+using System.Collections.Generic;
 using ParquetClassLibrary;
+using ParquetClassLibrary.Utilities;
 using Xunit;
 
 namespace ParquetUnitTests
 {
     public class AssemblyUnitTest
     {
         #region Values for Tests
+        /// <summary>This is the cannonical invalid version string used in serialization tests.</summary>
         private const string invalidDataVersion = "0.0.0";
+
+        /// <summary>
+        /// The highest <see cref="Range{EntityID}.Maximum"/> defined in <see cref="AssemblyInfo"/>
+        /// except for <see cref="AssemblyInfo.ItemIDs"/>.Maximum.
+        /// </summary>
+        public static readonly EntityID MaximumIDRangeUpperLimit = typeof(AssemblyInfo).GetFields()
```

Comment:
@WispyMouse I wanted to call special attention to this line.  This is the first complex Linq statement I've ever written.  While I've tested it and it appears to work as intended, I would appreciate any tips or insights you may have!

Reply:
The reason this is reproduced here is because I expect the implementation in AssemblyInfo may change, and I want the test to be independent of that implementation.